package.json

```diff
@@ -25,12 +25,10 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0",
-    "@types/glob": "^7.1.3",
-    "add": "^2.0.6",
-    "glob": "^7.1.4",
-    "global": "^4.4.0"
+    "glob": "^7.1.4"
   },
   "devDependencies": {
+    "@types/glob": "^7.1.3",
```

Comment:
I'm pretty sure this needs to stay in `dependencies`. If you clear out node_modules then perform a production only build typescript will fail to build (ex- `yarn clean && yarn install --production && yarn build`).